apps/playwright/utils/graphql.ts

```diff
@@ -0,0 +1,42 @@
+import { chromium } from "@playwright/test";
+import auth from "~/constants/auth";
+import { getAuthTokens } from "./auth";
+
+export type GraphQLResponse<K extends string, T> = {
+  [k in K]: T;
+};
+
+/**
+ * GraphQL Request
+ *
+ * Make a GraphQL request using the
+ * admin auth state by default.
+ */
+export async function graphqlRequest(
+  query: string,
+  variables?: Record<string, unknown>,
+  storageState?: string,
+) {
+  const browser = await chromium.launch();
```

Comment:
Yeah, that is a good thing to bring up. Originally, that was how things were happening and I really do agree that is how we should be doing it. This was an easy fix to get the auto refresh of our client since the access token is short lived and usually expires before the test is done. 

I should have written something about this so we could fix it but this was the easy fix. I think we could write something into the fixture to intercept refresh requests and update the auth storage state so it is always up to date. There are other ways but this is likely the best solution.